docs/Anjay_integrations/ESP32_integration/M5Stick.md

```diff
@@ -63,14 +72,24 @@ To connect the board:
     - In the **Device ID** field, type the endpoint name provided in the `nvs_config.csv`, e.g. `ESP32_test`.
     - In the **Security mode** section, select the **PSK** mode.
     - In the **Key identity** field, type the name provided in the `nvs_config.csv`, e.g. `ESP32_test`.
-    - In the **Key identity** field, type the identity provided in the `nvs_config.csv`, e.g. `ESP32_test`.
     - In the **Key** field, type the `psk` key provided in the `nvs_config.csv`.
     ![Device credentials step](images/add_mgmt_quick.png "Device credentials step")
 0. Click the **Add device** button and **Confirm** in the confirmation pop-up.
 0. In the **Connect your device** step, the server is waiting for the board to connect. You can now start connecting the device.
 
 ## Step 4: Flash the board and run device
 
+!!! tip
+        Before flashing the device you should erase the flash first to make sure you have correct settings:
+        === "Linux"
+            ``` linux
+                esptool.py erase_flash
+            ```
+        === "Windows"
+            ``` windows
+                esptool erase_flash
+            ```
+
 Use pre-built binaries to flash the board and provide credentials by flashing the NVS partition binary.
 
 0. Install the `esptool.py`:
```

Comment:
You need to put this step before the `tip` section ("Before flashing the device you should erase the flash...")